docs/topics/proc_configuring-your-application-to-use-spring-boot.adoc

```diff
@@ -47,6 +47,29 @@ Include the following properties in your `pom.xml` file to track the version of
 ----
 --
 
+. Specify the repositories containing {ProductShortName} {SpringBoot} Starters and the {SpringBoot} Maven Plugin.
```

Comment:
It is customary to end an ushering sentence to a figure with a semicolon.

Reply:
I agree, let me fix this too, then:)

Reply:
fixed:)